cmd/zfs_object_agent/zettaobject/src/pool.rs

```diff
@@ -54,6 +55,9 @@ lazy_static! {
     static ref LOG_CONDENSE_MULTIPLE: usize = get_tunable("log_condense_multiple", 5);
 
     static ref CLAIM_DURATION: Duration = Duration::from_secs(get_tunable("claim_duration_secs", 2));
+
+    // By default, retain metadata for as long as we would in a block-based pool
```

Comment:
```suggestion
    // By default, retain metadata for as long as we would retain Uberblocks in a block-based pool
```
(since block-based pools don't retain any other kind of metadata)